centreon-open-tickets/www/modules/centreon-open-tickets/centreon-open-tickets.conf.php

```diff
@@ -21,6 +21,10 @@
 
 require_once realpath(__DIR__ . '/../../../bootstrap.php');
 
+if (!isset($centreon_path) || empty($centreon_path)) {
```

Comment:
[`empty`](https://www.php.net/manual/fr/function.empty.php) already does [! `isset`](https://www.php.net/manual/fr/function.isset.php) 
```suggestion
if (empty($centreon_path)) {
```